draft-ietf-quic-multipath.md

```diff
@@ -541,6 +541,12 @@ both to avoid generating spurious stateless packets as specified in
 {{spurious-stateless-reset}} and to be able to acknowledge the
 last packets received from the peer as specified in {{ack-after-abandon}}.
 
+If a peer sends a PATH_ABANDON frame but never receives
+a corresponding PATH_ABANDON frame, it might not be able to remove path state.
+It is left to the implementation to handle this unexpected
+behavior as it does not impact interoperability. However, an endpoint SHOULD NOT
+remove path state earlier than 3 PTOs after sending the PATh_ABANDON frame.
```

Comment:
```suggestion
behavior as it does not impact interoperability. If the endpoint is no longer 
willing to process the issued connection IDs for the abandoned path,
it MAY close the connection, but SHOULD wait at least 3 PTOs after
sending the PATh_ABANDON frame.
```

Reply:
Yes, that works. It avoids saying creative things, and it sends the right message to developers: reply by ABANDON to ABANDON, or risk losing the connection. Note the typo "PATh_ABANDON".

Reply:
I believe it works.

Reply:
Yes, this works for me.